### PR TITLE
Update PostgresqlDriver.php

### PR DESCRIPTION
**Pull request by jdeproost**
Update PostgresqlDriver to support non default port 5432. Same principal as MySQLi driver is used to get the database hostname and port from the host connection string (databasehost:port)

### CODE REVIEW

File: src/Joomla/Database/Postgresql/PostgresqlDriver.php [hunk 1]
```diff
@@ -112,9 +113,43 @@ public function connect()
...
+			{
+			$this->options['host'] = 'localhost';
+			}
+			}
+			// No port annotation (:) found, setting port to default PostgreSQL port 5432
+			else
+			{
+			$this->options['port'] = '5432';
+			}
+		}
```
Comment (dongilbert) [inline src/Joomla/Database/Postgresql/PostgresqlDriver.php:149]: Indention in this section looks a bit off, maybe you're mixing tabs and spaces?

File: src/Joomla/Database/Postgresql/PostgresqlDriver.php [hunk 2]
```diff
@@ -112,9 +113,43 @@ public function connect()
...
+		* have to extract it from the host string (if povided).
+		*/
+		
+		// Check for empty port
+		if (!($this->options['port']))
+		{
+		// Port is empty or not set via options, check for port annotation (:) in the host string
+		$tmp = substr(strstr($this->options['host'], ':'), 1);
+		
+			if (!empty($tmp))
```
Comment (dongilbert) [inline src/Joomla/Database/Postgresql/PostgresqlDriver.php:127]:
``` php
if (!empty($tmp) && is_numeric($tmp))
{
    $this->options['port'] = $tmp;
}
```

Comment (dongilbert) [inline src/Joomla/Database/Postgresql/PostgresqlDriver.php:127]: Actually that might not apply, since it's hard to follow the indentations here.

**Comment by jdeproost** [comment]
Fixed the indentation

**Comment by dongilbert** [comment]
Merged via b0727287f2e9ec079957fb94518f70535ea54734

Had to fix a code style issue. Thanks for this!
